test/xapi/implementation_test.rb

```diff
@@ -50,34 +50,13 @@ def test_implementation
   def test_implementation_with_extra_files # including dotfiles and files in subdirectories
     problem = Xapi::Problem.new('one', FIXTURE_PATH)
     implementation = Xapi::Implementation.new('fake', "[URL]", problem, FIXTURE_PATH)
-    readme = <<-README
-# One
-
-This is one.
-
-* one
-* one again
-
-* one hint
-* one more hint
-
-## Source
-
-The internet. [http://example.com](http://example.com)
-
-## Submitting Incomplete Problems
-It's possible to submit an incomplete solution so you can see how others have completed the exercise.
-
-    README
```

Comment:
Do we still have a test that says that the implementation returns the correct file contents?